src/Widgets/SettingsPopover.vala

```diff
@@ -113,6 +113,11 @@ public sealed class Terminal.SettingsPopover : Gtk.Popover {
             active = Application.settings.get_boolean ("natural-copy-paste")
         };
 
+        var audible_bell_button = new Granite.SwitchModelButton (_("Audible Bell")) {
+            description = _("Send an event alert for invalid input or multiple possible completions (subject to system sound settings)"),
```

Comment:
In the description text in System Settings we say "notify" here. Also use the same format we use in dialogs etc to describe System Settings locations

```suggestion
            description = _("Notify for invalid input or multiple possible completions (subject to System Settings → Sound)"),
```